src/profile.js

```diff
@@ -135,9 +171,9 @@ import {withRouter} from 'react-router-dom'
               type="int"
               name="birthday_yr"
               min="1920"
-              max="2100"
               onChange={this.handleChange}
```

Comment:
I think you still need a max here so user doesn't input a year that's after today's year. you can set max to today's year

Reply:
Users below 18 can't register, so if they put a date greater than today's date then it will still block them from registering.